services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -227,21 +223,21 @@ async def get_item_from_collection(self, request: Request):
         try:
             item = await self.client.get_item(
                 item_id=self.request_ids["item_id"],
-                collection_id=f"{self.request_ids['owner_id']}_{self.request_ids['collection_id']}",
+                collection_id=f"{self.request_ids['owner_id']}_{self.request_ids['collection_id'][0]}",
```

Comment:
Why you access index 0 now? What's stored in other indexes that previous wasn t?

Reply:
This is because the collection_id is now a list and not a string (because we are now able to use several collections for the search endpoint). 
In some use cases such as the creation of a new item in a collection, there is only one collection specified in the list "collection_id" 
(because we create an item that will belong to only one collection), that's why just have to get the first (and only) element of the list